### PR TITLE
fix: Fix invite flow to existing team

### LINKED ISSUE #10949: [CAL-2388] Non Cal.com Users Cannot Create Account via Direct Team Invite
Accounts that get invited directly from a team (i.e. add email, send invite) **that do not have an account created already** receive the following error which was previously reported

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2388](https://linear.app/calcom/issue/CAL-2388/non-calcom-users-cannot-create-account-via-direct-team-invite)</sub>

### PR DESCRIPTION
**Pull request by sean-brydon**
Fixes invite flow for existing teams

### CODE REVIEW

File: apps/web/pages/api/auth/signup.ts [hunk 1]
```diff
@@ -79,11 +79,11 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     }
     if (foundToken?.teamId) {
       const teamUserValidation = await validateUsernameInTeam(username, userEmail, foundToken?.teamId);
-      return validationResponse(userEmail, teamUserValidation);
+      validationResponse(userEmail, teamUserValidation);
```
[the remark this answers is not in the document]
Comment (sean-brydon) [reply apps/web/pages/api/auth/signup.ts:82]: The invalid response is returned https://github.com/calcom/cal.com/pull/10951/files#diff-5685bc8d87e36ef45d4423f5f5bedcd2de3a4654b96a17121fbe9c20caf58790L51

File: apps/web/pages/api/auth/signup.ts [hunk 2]
```diff
@@ -79,11 +79,11 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
...
     }
   } else {
     const userValidation = await validateUsername(username, userEmail);
-    return validationResponse(userEmail, userValidation);
+    validationResponse(userEmail, userValidation);
```
Comment (emrysal) [inline apps/web/pages/api/auth/signup.ts:86]: Don't we have to get the response here, to return if it is non-void?

Comment (emrysal) [inline apps/web/pages/api/auth/signup.ts:86]: Don't we have to get the response here, to return if it is non-void?

**Review by emrysal — APPROVED** [review]
LGTM

**Review by emrysal — APPROVED** [review]
LGTM

**Review by Udit-takkar — APPROVED** [review]
LGTM

**Pull request merged**